docs/46_local_dev.md

```diff
@@ -0,0 +1,13 @@
+If you'd like to develop these docs, the follow commands should get you started locally -- with hot-reloading too :)
```

Comment:
@kabilar Note for local dev spin-up

Reply:
Thank you @aaronkanzer.  This works well.